mail/header.go

```diff
@@ -211,6 +212,61 @@ func (p *headerParser) parseMsgID() (string, error) {
 	return left + "@" + right, nil
 }
 
+func (p *headerParser) parseListCommand() (*url.URL, error) {
+	if !p.skipCFWS() {
+		return nil, errors.New("mail: malformed parenthetical comment")
+	}
+
+	// Consume a potential newline + indent.
+	p.consume('\r')
+	p.consume('\n')
+	p.skipSpace()
```

Comment:
I don't think the input values ever contain newlines. These are stripped by the header parser when unfolding the header fields.